resources/migrations/000_migrations.yaml

```diff
@@ -14354,8 +14354,45 @@ databaseChangeLog:
               WHEN MATCHED THEN
                 UPDATE SET f.base_type = updates.base_type;
 
+  - changeSet:
+      id: v47.00-002
+      author: calherries
+      comment: Added 0.47.0 - Add json_unfolding column to metabase_field
+      changes:
+        - addColumn:
+            columns:
+            - column:
+                remarks: 'Enable/disable JSON unfolding for a field'
+                name: json_unfolding
+                type: boolean
+                constraints:
+                  nullable: true
+            tableName: metabase_field
+
+  - changeSet:
```

Comment:
To preserve unfolding of existing fields, this backfills the new `metabase._field.json_unfolding` column to equal true if `metabase_database.details.json-unfolding` is true or nil.